pkg/github/models/converted_note_to_issue_issue_event.go

```diff
@@ -4,7 +4,7 @@ import (
     i878a80d2330e89d26896388a3f487eef27b0a0e6c010c493bf80be1452208f91 "github.com/microsoft/kiota-abstractions-go/serialization"
 )
 
-// ConvertedNoteToIssueIssueEvent converted Note to Issue Issue Event
+// ConvertedNoteToIssueIssueEvent converted Note to Issue IssueEvent
```

Comment:
Here, the `converted Note to Issue` is an `IssueEvent` and it makes sense to make it possible to parse by removing the space ...